Alignment/OfflineValidation/macros/PlotAlignmentValidation.C

```diff
@@ -429,12 +479,13 @@ void PlotAlignmentValidation::plotSS( const std::string& options, const std::str
       continue;
 
     // Skips plotting too high layers
-    if (plotLayerN > numberOfLayers[iSubDet-1]) {
+    if (plotLayerN > numberOfLayers(1, iSubDet)) {
       continue;
     }
 
     int minlayer = plotLayers ? 1 : plotLayerN;
-    int maxlayer = plotLayers ? numberOfLayers[iSubDet-1] : plotLayerN;
+    int maxlayer = plotLayers ? numberOfLayers(1, iSubDet) : plotLayerN;
+    int maxlayerphase0 = plotLayers ? numberOfLayers(0, iSubDet) : plotLayerN;
```

Comment:
@hroskes can you elaborate on the logic to distinguish between phase-0 and phase-1 ?